src/nupic/utils/Random.hpp

```diff
@@ -122,13 +122,6 @@ class Random : public Serializable  {
     pop.resize(nChoices); //keep only first nChoices, drop rest
     return pop;
   }
-  //compatibility method for Py-bindings, //TODO remove with SWIG 
```

Comment:
I'm removing old wrapper for Random::sample, 
updating Sdr.cpp and py bindings.